tests/php/ORM/Search/SearchContextTest.php

```diff
@@ -138,16 +138,36 @@ public function testUserDefinedFiltersAppearInSearchContext()
     public function testUserDefinedFieldsAppearInSearchContext()
     {
         $company = SearchContextTest\Company::singleton();
+        $searchName = $company->getGeneralSearchFieldName();
```

Comment:
The issue this is fixing is NOT caused by the changed to CustomMethods::getExtraMethodConfig() above

Fixes issue

`1) SilverStripe\ORM\Tests\Search\SearchContextTest::testUserDefinedFieldsAppearInSearchContext
Failed asserting that two objects are equal.
--- Expected
+++ Actual
@@ @@
             'failover' => null
             'customisedObject' => null
             'objCache' => []
-            'extension_instances' => null
+            'extension_instances' => [...]`

Reply:
It's hard to tell what you've actually changed here, other than adding a whole load of assertions that weren't there before.
What was the root cause here and what specific change fixes it, vs just adding new tests?

Reply:
I don't know what the root cause is, and I'm not sure if it's an actually issue

This specific unit test doesn't care about extension_instances, what it's actually testing is that the scaffolded search fields match Company.$searchable_fields

Reply:
Have changed solution